contributing.md

```diff
@@ -8,13 +8,15 @@ below:
 ## Subject
 *Description*
 ### Learning
-* Thing 1 &mdash; Short description of Thing 1
-* Thing 2 &mdash; Short description of Thing 2
+* Thing 1 &mdash; Short description of Thing 1.
+* Thing 2 &mdash; Short description of Thing 2.
 ### Culture
-* Thing 3 &mdash; Short description of Thing 3
+* Thing 3 &mdash; Short description of Thing 3.
 
 Those are `&mdash;` 
 
+Note that every description must end in some sort of punctiation. 
```

Comment:
You have a typo here, stabs :P

Reply:
Also not sure it's necessary, but not opposed to adding it anyway.

Reply:
Ah will fix. It's not about it being necessary, it's about being consistent...